composer.json

```diff
@@ -10,6 +10,7 @@
     }
   },
   "require": {
+    "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Does this project support PHP 8.2 yet?

Running

```shell
git grep 8.2
```

on current `1.x` yields nothing.

Reply:
Not yet, we are currently working on it for the core https://github.com/pimcore/pimcore/pull/14080